examples/_pages/examples_test.ts

```diff
@@ -0,0 +1,21 @@
+// import { join } from "@std/path";
+// import { walk } from "@std/fs";
+// import { assertEquals, assertNotMatch } from "@std/assert";
+
+// const decoder = new TextDecoder();
+
+// Deno.test("Check examples", async (t) => {
+//   for await (const item of walk("./examples/")) {
+//     const path = join("examples", item.name);
+
+//     if (!path.endsWith(".ts")) continue;
+
+//     await t.step("Check graph: " + path, async () => {
+//       const result = await new Deno.Command(Deno.execPath(), {
+//         args: ["info", path],
+//       }).output();
+//       assertEquals(result.code, 0);
+//       assertNotMatch(decoder.decode(result.stdout), /\(resolve error\)/);
+//     });
+//   }
+// });
```

Comment:
Should this be here? Seems like we should either write tests or delete the file.

Reply:
Ah dammit. I completely forgot about the tests. Will double check tomorrow whether they work or not!

Reply:
fixed!